app/Resources/views/topedits/result_article.html.twig

```diff
@@ -1,20 +1,26 @@
-G{% extends 'base.html.twig' %}
+{% extends 'base.html.twig' %}
 
 {% block body %}
 
 
     <div class="panel panel-primary" style="text-align:center">
         <div class="panel-heading">
             <p class="xt-heading-top" >
-                <a href="//en.wikipedia.org/wiki/List_of_space_stations">List of space stations</a>
-                <span style="padding-left:10px;" > &bull;&nbsp; en.wikipedia.org </span>
+                <a href="{{ article.fullurl }}">
+                    {% if article.pageprops.displaytitle is defined %}
+                        {{ article.pageprops.displaytitle|raw }}
+                    {% else %}
+                        {{ article.title }}
+                    {% endif %}
+                </a>
+                <span style="padding-left:10px;" > &bull;&nbsp; {{ project }} </span>
             </p>
         </div>
 
         <div class="panel-body xt-panel-body-top" >
             <p>
```

Comment:
Done. (I was just basing it on the HTML that was already in that file, but now you've done great work on the structure I'll ensure to conform to that from here on.)